kong/runloop/handler.lua

```diff
@@ -204,6 +211,14 @@ local function csv(s)
 end
 
 
+local function try_incr_counter(key)
+  -- key must be DECLARATIVE_ROUTERS_REBUILD_COUNT_KEY or DECLARATIVE_PLUGINS_REBUILD_COUNT_KEY
+  local _, err = kong_shm:incr(key, 1, 0)
+  if err then
+    log(ERR, "failed to increase router/plugins rebuild counter: ", err)
+  end
+end
+
```

Comment:
If we don't want this function to be used for other keys, please use a more specific name, and add `assert` at top of the function body.